qa/tasks/crimson_objectstore_tool.py

```diff
@@ -0,0 +1,536 @@
+"""
+crimson_objectstore_tool - Test of crimson-objectstore-tool utility
```

Comment:
Can we reuse the existing `qa/tasks/ceph_objectstore_tool.py`? IIUC, the main difference is the tool binary name which could be defined based on a new supported parameter? Something like:
```
- objectstore_tool:
    crimson_objectstore_tool: true #use crimson-objectstore-tool
    objects: 5
    pgnum: 8
```
and a corresponding:
```
CRIMSON = config.get('crimson_objectstore_tool', false)
```
Any unsupported features could be also skipped by introducing a `SKIP_IF_CRIMSON` method. We use similar approach with all `/src/test/..` unittests.